.lipublish/publish.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
will change README to include all of the info on how to publish if the setup looks ok